vagrant-manager241

geggo98 opened the pull request:
Version 241 for vagrant manager. This old version doesn't contain some regressions of version 244.

* [Issue 111](lanayotech/vagrant-manager#111)
* [Issue 107](lanayotech/vagrant-manager#107)

vitorgalvao commented on Casks/vagrant-manager241.rb:
```diff
@@ -0,0 +1,13 @@
+
+  # github.com is the official download host per the vendor homepage
+  url "https://github.com/lanayotech/vagrant-manager/releases/download/#{version}/vagrant-manager-#{version}.dmg"
+  appcast 'https://github.com/lanayotech/vagrant-manager/releases.atom'
```
Remove the appcast (since it would defeat the purpose of version locking).

vitorgalvao commented:
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

geggo98 commented:
Squashed the commits (two tries, was my first time...) and removed the app cast.

@vitorgalvao Please check the new version carefully, I might have introduced some errors in the squashing progress. Please tell me if I should fix something. And thanks for the helpful link!

vitorgalvao commented:
Looks good!

vitorgalvao commented:
Looks good, thank you!
